columnflow/production/normalization.py

```diff
@@ -331,6 +340,8 @@ def normalization_weights_init(self: Producer) -> None:
     Initializes the normalization weights producer by setting up the normalization weight column.
     """
     self.produces.add(self.weight_name)
+    if self.allow_stitching:
+        self.produces.add(f"{self.weight_name}_incl")
```

Comment:
Done, but I had to move a small code snippet from the `require` function to the `init` function, since otherwise the `get_inclusive_dataset` would give an error.  Is this fine?